akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -328,11 +328,21 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
   val largeMessageChannelEnabled =
     !largeMessageDestinations.wildcardTree.isEmpty || !largeMessageDestinations.doubleWildcardTree.isEmpty
 
+  private val priorityMessageDestinations =
+    WildcardIndex[NotUsed]()
+      // this comes from remoting so is semi-ok to be hardcoded here
```

Comment:
WDYM by semi-ok? It should be completely OK I think. Especially since there is no ordering requirement between Watch and other messages as it is a system message. (At least I am not aware of any such requirement)